### DISCUSSION TITLE
When inside a method call should its arguments have priority in the suggestions list?

### DISCUSSION THREAD

**Discussion by MangelMaxime** [ionide/ionide-vscode-fsharp#2054] [Ideas]
Currently, the method arguments are lost in the middle of all the suggestions making it difficult to discover the available arguments.

https://github.com/user-attachments/assets/562c6892-09fd-426e-ad5c-01e51b9a0bea

**Comment by TheAngryByrd** [comment]
Yeah anything that's local should probably have priority.
